src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -2,8 +2,8 @@ openapi: 3.0.3
 info:
```

Comment:
Aj ked ide len o whitespace, skusme taketo zmeny nedavat do PR kde to nie je relevantne, lebo to natahuje kod co musim citat.